quiz.js

```diff
@@ -37,9 +50,23 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update({
+ name: 'Stark' }, { $push: { member: 'Ned Stark'}});
 
+db.houses.update({
+ name: 'Stark' }, { $push: { member: 'Arya Stark'}});
 
+db.houses.update({
+ name: 'Stark' }, { $push: { member: 'Sansa Stark'}});
 
+db.houses.update({
+ name: 'Targaryen' }, { $push: { member: 'Viserys Targaryen'}});
+
+db.houses.update({
+ name: 'Targaryen' }, { $push: { member: 'Daenerys Targaryen'}});
+
+db.houses.update({
+ name: 'Arryn' }, { $push: { member: 'Jon Arryn'}});
 
 // your code ends here
```

Comment:
This is all correct. However, this approach could get tedious if we were trying to add many members; Mongo's [$each](http://docs.mongodb.org/manual/reference/operator/update/each/) operator (shown below) would be a lot easier in that kind of scenario.
```
db.houses.update({name: 'Stark'}, { $push: { people: { $each: 
  [
    'Ned Stark', 
    'Arya Stark', 
    'Sansa Stark'
  ]
}}});
```